docs/docs/installation/INSTALLATION.md

```diff
@@ -96,6 +98,8 @@ volumes:
 ```
 
 > For more details, please refer to the [Volumes section of the configuration guide](CONFIGURATION.md#volumes).
+
+> If you are migrating an existing Data Center product to Kubernetes, use the values of your product's shared home. See [Migration guide](MIGRATION.md).
```

Comment:
Can we move this to the top of this section. There are already a few caveats here which might be confusing i.e.

Reply:
@nanux Im thinking it would be nice to highlight this using mkdocs - only instead of a ⚠️ symbol we use a ℹ️  symbol i.e

![image](https://user-images.githubusercontent.com/409063/128458053-9929764a-e7cf-4943-8c13-10f010d9f99f.png)

Can this be done?

Reply:
I moved it to the top of the section. Not sure what you mean - highlighting the migration comment? Or change the support disclaimer's warning? or something else?

Reply:
<img width="965" alt="Screen Shot 2021-08-06 at 5 02 15 pm" src="https://user-images.githubusercontent.com/416238/128470063-b2dbfe97-3c1e-4ccf-b4d4-facad0b2c36e.png">

I believe @errcode1202 meant something like this ^. I have pushed the change (see the latest commit).